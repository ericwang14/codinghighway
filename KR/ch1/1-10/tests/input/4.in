\
*
hi!			this is another test
\\\\\\\\\\\\\\
backslash escapes are 	evil!

	
	

