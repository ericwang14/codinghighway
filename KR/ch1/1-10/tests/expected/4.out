\\
*
hi!\t\t\tthis is another test
\\\\\\\\\\\\\\\\\\\\\\\\\\\\
backslash escapes are \tevil!

\t
\t

